ui/widgets/mini-profile/src/index.ts

```diff
@@ -0,0 +1,18 @@
+import 'systemjs-webpack-interop/auto-public-path';
+import {
+  IntegrationRegistrationOptions,
+  IWidgetConfig,
+} from '@akashaproject/ui-awf-typings/lib/app-loader';
+
+/**
+ * All widgets must export an object like this:
+ */
+export const register: (opts: IntegrationRegistrationOptions) => IWidgetConfig = opts => ({
+  mountsIn: opts.layoutConfig?.widgetSlotId,
+  loadingFn: () => import('./MiniProfileWidget'),
+  activeWhen: (location, pathToActiveWhen) =>
+    pathToActiveWhen('/social-app/post/', false)(location),
```

Comment:
is there a way to get this path from the routing plugin here?

Reply:
hmm, let me check

Reply:
looks like i can't trick it to make it work, the moment the activeWhen is called, it can happen that not all integrations are stored in the routing plugin...